elf/perf_unsupported.go

```diff
@@ -10,6 +10,8 @@ func InitPerfMap(b *Module, mapName string, receiverChan chan []byte) (*PerfMap,
 	return nil, fmt.Errorf("not supported")
 }
 
+func (pm *PerfMap) SetTimestampFunc(timestamp func(*[]byte) uint64) {}
```

Comment:
does it compile?

It should have a "return" statement because the function does not return void.